### PR TITLE
Yro/yt callback bugfix

### PR DESCRIPTION
**Pull request by yro**
### Description
Small updates to eliminate common daemon problems causing instability in the processes.

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review: @muhammad-ammar @Qubad786 

FYI: @edx/educator-neem @edx/educator-mallow 

### Testing
- [ ] Unit tests

### CODE REVIEW

File: control/veda_file_discovery.py [hunk 1]
```diff
@@ -180,11 +181,14 @@ def get_or_create_course(self, course_id, course_hex=None):
...
                     course.save()
             else:
+                course_code = generate_course_id(org=course_key.org, number=course_key.course)
+                semester = datetime.datetime.now().year
                 course_name = '{org} {number}'.format(org=course_key.org, number=course_key.course)
                 course = Course.objects.create(
                     course_name=course_name,
-                    institution=course_key.org,
-                    edx_classid=course_key.course,
+                    institution=course_code[0:3],
```
Comment (Qubad786) [inline control/veda_file_discovery.py]: Why shouldn't we put `course_key.org` ?

File: control/veda_file_discovery.py [hunk 2]
```diff
@@ -180,11 +181,14 @@ def get_or_create_course(self, course_id, course_hex=None):
...
+                    edx_classid=course_code[3:8],
```
Comment (Qubad786) [inline control/veda_file_discovery.py]: same as above why shouldn't it be `course_key.course`?

**Comment by Qubad786** [comment]
@yro [this](https://github.com/edx/edx-video-pipeline/blob/56b28ce31c713ae11ee6fce46ad486fe8a7887a0/control/veda_file_ingest.py#L270-L272) is where we should be using your util.

**Comment by Qubad786** [comment]
@yro you can clean file discovery.

**Comment by yro** [comment]
@Qubad786 This is all set. Feel free to merge if you're happy with it and pressed for time.

File: VEDA_OS01/utils.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 """
 Common utils.
 """
+import uuid
```
Comment (Qubad786) [inline VEDA_OS01/utils.py]: unused import.

**Comment by yro** [comment]
@Qubad786 The travis build seems to be having some issues. I think it's an issue on travis' side, not mine. If you need to merge, go ahead, but this isn't a breaking change and can go out on the next release.

**Comment by Qubad786** [comment]
Yeah, I see. There has been outage yesterday: https://www.traviscistatus.com/.

**Comment by Qubad786** [comment]
@yro since we don't have tests coverage for YT callback module and this has urgency, I am going to merge this as is.

**Comment by Qubad786** [comment]
Build is green.

**Pull request merged**